backend/src/main/scala/cromwell/backend/RuntimeAttributeDefinition.scala

```diff
@@ -42,7 +41,7 @@ object RuntimeAttributeDefinition {
     // e.g. `gcp: userDefinedObject` to find out what its runtime value is.
     // The type system informs us of this because a `WomExpression` in `unevaluated`
     // cannot be safely read as a `WomObject` with a `values` map until evaluation
-    evaluated.map(e => applyPlatform(e, platform))
+    evaluated.map(e => applyPlatform(e, platform)).map(e => applyOverridesFromInputs(e, evaluatedInputs))
```

Comment:
100% optional naming normalization
```suggestion
    evaluated.map(e => applyOverridesFromPlatform(e, platform)).map(e => applyOverridesFromInputs(e, evaluatedInputs))
```